sdk/lib/opentelemetry/sdk/configurator.rb

```diff
@@ -148,14 +148,41 @@ def configure_span_processors
       end
 
       def default_span_processor
-        Trace::Export::SimpleSpanProcessor.new(
+        Trace::Export::SimpleSpanProcessor.new(configured_exporter)
+      end
+
+      def configured_exporter
```

Comment:
nitpick: `exporter_from_env` or some variation of that  is probably a more accurate name.